tests/HTMLPurifierTest.hack

```diff
@@ -148,14 +228,17 @@ class HTMLPurifierTest extends HackTest {
 			new Token\HTMLPurifier_Token_Text("Bold"),
 			new Token\HTMLPurifier_Token_End("b", dict[]),
 		];
-
+		\var_dump($tokens);
 		expect($tokens)->toHaveSameContentAs($expected_tokens);
 		echo "finished.\n";
 	}
 
 	public function testStrategies(): void {
 		echo "\nrunning testStrategies()...";
 		$config = HTMLPurifier\HTMLPurifier_Config::createDefault();
+		$policy = HTMLPurifier\HTMLPurifier_Policy::fromDefault();
+		// This is weird, but based off of how we define config in HTMLPurifier constructor
+		$config = $policy |> $$->configPolicy($config);
```

Comment:
Same for this one. I agree that this setup is pretty confusing and think the pipe makes it even moreso.

Reply:
Same as the other comment